### PR TITLE
Add Nexus webhook receiver 

### PR DESCRIPTION
**Pull request by SomtochiAma**
Fixes: #126 

Signed-off-by: Somtochi Onyekwere <somtochionyekwere@gmail.com>

### CODE REVIEW

File: docs/spec/v1beta1/receiver.md [hunk 1]
```diff
@@ -229,6 +230,23 @@ spec:
...
+  name: nexus-receiver
+  namespace: default
+spec:
+  type: nexus
+  secretRef:
+    name: webhook-token
+  resources:
+    - kind: ImageRepository
+      name: webapp
+```
```
Comment (stefanprodan) [inline docs/spec/v1beta1/receiver.md:248]: Please add the note here like we do for Harbor and mention the header we use `X-Nexus-Webhook-Signature`

**Review by stefanprodan — CHANGES_REQUESTED** [review]

File: docs/spec/v1beta1/receiver.md [hunk 2]
```diff
@@ -247,6 +247,10 @@ spec:
       name: webapp
 ```
 
+Note that you have to fill in the generated token as the secret key when creating the Nexus Webhook Capability.
+See [nexus documentation](https://help.sonatype.com/repomanager3/webhooks/enabling-a-repository-webhook-capability)
```
Comment (stefanprodan) [inline docs/spec/v1beta1/receiver.md:251]: move this link on Nexus Webhook Capability please.

**Review by stefanprodan — CHANGES_REQUESTED** [review]

File: docs/spec/v1beta1/receiver.md (hunk 2 above)
Comment (SomtochiAma) [reply docs/spec/v1beta1/receiver.md:251]: I should remove it? Or move it somewhere else?

Comment (stefanprodan) [reply docs/spec/v1beta1/receiver.md:251]: `[Nexus Webhook Capability](https://help.sonatype.com/repomanager3/webhooks/enabling-a-repository-webhook-capability)`

**Review by stefanprodan — APPROVED** [review]
LGTM

**Pull request merged**